packages/config/src/integrations.ts

```diff
@@ -62,8 +53,9 @@ export const mergeIntegrations = async function ({
 
       return {
         slug: integration.slug,
-        version: integration.hostSiteUrl,
```

Comment:
🙃

Reply:
@serhalp wait gimme more info

Reply:
Just confused about previously setting a field called "version" to a URL!